packages/core/src/schema/models/providerLogs.ts

```diff
@@ -35,6 +35,7 @@ export const providerLogs = latitudeSchema.table('provider_logs', {
       onUpdate: 'cascade',
     }),
   model: varchar('model'),
+  finishReason: varchar('finish_reason').notNull().default('stop'),
```

Comment:
We want to save what was the AI reason to finish. By default we set `stop` because is the most common reason